### ISSUE TITLE
Why stylelint doesn't works?

### ISSUE THREAD

**Issue by jon9090** [stylelint/stylelint#8138]
### What minimal example or steps are needed to reproduce the bug?

I inititilze empty project by running this command:  `npm init -y && npm init stylelint`
```
> k@1.0.0 npx
> create-stylelint

✔ Created .stylelintrc.json.
✔ Installed packages.

You can now lint your CSS files using:
npx stylelint "**/*.css"

We recommend customizing Stylelint:
https://stylelint.io/user-guide/customize/
```

![Image](https://github.com/user-attachments/assets/f33616c5-3473-4db1-92e0-0849a69c51bb)

The command: `npx stylelint "*.css" produces nothing and no output.

And vscode doesn't throw errors, you can see the picture.

what can it be? did I miss something here?

![Image](https://github.com/user-attachments/assets/c881a85e-fc94-4ce1-9411-306a983aa35d)

**Comment by jeddy3** [comment]
@jon9090 Thanks for the report.

`display: "bla` would be checked by the [`declaration-property-value-no-unknown`](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown/) rule, which we [added in 15.0.0](https://stylelint.io/migration-guide/to-15#added-declaration-property-value-no-unknown-rule) but didn't turn on in the standard config. We're [discussing](https://github.com/stylelint/stylelint/issues/8133) turning it on soon, though.

In the meantime, you can turn it on yourself:

```diff json
{
  "extends": ["stylelint-config-standard"],
  "rules": {
+   "declaration-property-value-no-unknown": true
    ..
   }
}
```

Or you can test Stylelint is working using:

```css
* {}
```

As the [block-no-empty](https://stylelint.io/user-guide/rules/block-no-empty) is [turned on](https://stylelint.io/user-guide/rules/#empty).

Feel free to reopen the issue if this doesn't resolve it for you.
